Cargo.toml

```diff
@@ -3,6 +3,11 @@ name = "openmls"
 version = "0.2.0"
 authors = ["Raphael Robert"]
 edition = "2018"
+description = "This is a PoC Rust implementation of the Messaging Layer Security (MLS) protocol based on draft 9+."
```

Comment:
I'd not call it PoC but something like WiP or whatever. But it's supposed to be a production library. It's just not there yet.